traffic_ops/testing/api/v2/profiles_test.go

```diff
@@ -210,12 +210,29 @@ func GetTestProfiles(t *testing.T) {
 			t.Errorf("cannot GET Profile by name: %v - %v", err, resp)
 		}
 		profileID := resp[0].ID
-
-		resp, _, err = TOSession.GetProfileByParameter(pr.Parameter)
-		if err != nil {
-			t.Errorf("cannot GET Profile by param: %v - %v", err, resp)
+		if len(pr.Parameters) > 0 {
+			parameter := pr.Parameters[0]
+			respParameter, _, err := TOSession.GetParameterByName(*parameter.Name)
+			if err != nil {
+				t.Errorf("Cannot GET Parameter by name: %v - %v", err, resp)
```

Comment:
This `resp` is the wrong response here. Also, I don't think you need to print the response here, just the error is fine.

Reply:
removed the response in error statement